Marlin/configuration_store.cpp

```diff
@@ -324,9 +344,9 @@ void Config_StoreSettings()  {
     EEPROM_WRITE_VAR(i, dummy);
   }
 
-  char ver2[4] = EEPROM_VERSION;
   int j = EEPROM_OFFSET;
-  EEPROM_WRITE_VAR(j, ver2); // validate data
+  EEPROM_WRITE_VAR(j, version);
+  EEPROM_WRITE_VAR(j, eeprom_checksum);
```

Comment:
Delete EEPROM_WRITE_VAR(j, eeprom_checksum);
and write stored_checksum
EEPROM_WRITE_VAR(j, stored_checksum);